server/libs/infra/src/migrations/1683734723680-RemovePartnerIdColumn.ts

```diff
@@ -0,0 +1,30 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class RemovePartnerIdColumn1683734723680 implements MigrationInterface {
+    name = 'RemovePartnerIdColumn1683734723680'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Since this is within the same PR, we should be able to revert/drop both of these migrations, and then re-generate a single/unified one where the table is created how we want it from the get-go. There are some scripts in the server/package.json for undoing migrations and regenerating them.

TLDR - this PR should have a single migration file that creates the partner table.